app/src/main/java/com/ichi2/apisample/helper/AnkiDroidHelper.java

```diff
@@ -135,6 +135,98 @@ public Long addNewCustomModel(String modelName, String[] fields, String[] cards,
         return getApi().addNewCustomModel(modelName, fields, cards, qfmt, afmt, css, null, null);
     }
 
+    public boolean checkCustomModel(long modelId, String[] fields, String[] cards, String[] qfmt, String[] afmt, String css) {
```

Comment:
It probably makes sense to add some short work description for these methods, since understanding the code, if necessary, may take some time.

Reply:
Added doc comments